_data/glossary/persistent-volume-claim.yaml

```diff
@@ -0,0 +1,15 @@
+id: persistent-volume-claim
+name: Persistent Volume Claim
+full-link: /docs/concepts/storage/persistent-volumes/
+related:
+- persistent-volume
+- statefulset
+- deployment
+- pod
+tags:
+- core-object
+- storage
+short-description: >
+  A PersistentVolumeClaim (PVC) is used to claim storage resources defined in a PersistentVolume (PV) so that it can be mounted as a volume in a container.
```

Comment:
Drop "A PersistentVolumeClaim (PVC) is" to avoid repeating the term.

Reply:
And to clarify, start your (new) fragment with "Claims the storage resources that are defined by a PersistentVolume object ..." (no need to include "is used," which adds no meaning)